st2common/st2common/conf/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 The StackStorm Authors.
```

Comment:
Was this file really created in 2021?

Reply:
Yes I originally committed this on Jun 12 2021. But, since it's getting merged in 2023, I've updated the copyright.